app/utils/DateFormatter.scala

```diff
@@ -40,7 +40,7 @@ class DateFormatter @Inject()(config: FrontendAppConfig, languageUtils: Language
   }
 
   def formatDate(date: JodaDate)(implicit messages: Messages): String = {
-    languageUtils.Dates.formatDate(date)
+    languageUtils.Dates.formatDate(JavaDate.of(date.getYear, date.getMonthOfYear, date.getDayOfMonth))
```

Comment:
I have a feeling there's a java date -> joda date -> java date conversion going on here. Can just get rid of all the joda stuff